### PR TITLE
newspaper3k nodes integration

### PR DESCRIPTION
**Pull request by Haradai**
Two nodes that make use of the newspaper3k library for scraping articles and crawling many pages.

### CODE REVIEW

File: integrations/newspaper3k.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+            et              Estonian
+            ja              Japanese
+            be              Belarusian
+
+        :param metadata: (False by default) Wether to get article metadata.
+        :param keywords: (False by default) Wether to save the detected article keywords as document metadata.
+        :param summary: (False by default) Wether to summarize the document (through nespaper3k) and save it as document metadata.
+        :param path: (None by default) Path where to store the downloaded articles html, if None, not downloaded.
+```
+**In standalone:**
```
Comment (TuanaCelik) [inline integrations/newspaper3k.md:182]: Standalone

File: integrations/newspaper3k.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+        "negative":["facebook","instagram"]
+    },
+    keep_links = False,
+    metadata=True,
+    summary=True,
+    keywords=True,
+    path = "articles")
+```
+
+**In a pipeline:**
```
Comment (TuanaCelik) [inline integrations/newspaper3k.md:201]: In a Pipeline

**Comment by Haradai** [comment]
Allright in principle I should have deleted the DS_Store and changed the capitalizations🚀

**Review by TuanaCelik — APPROVED** [review]

**Comment by TuanaCelik** [comment]
@Haradai thank you for your contribution, your integration is now live: https://haystack.deepset.ai/integrations/newspaper3k 
🚀

**Pull request merged**